eng/pipelines/templates/stages/platform-matrix.json

```diff
@@ -0,0 +1,45 @@
+{
+  "displayNames": {
+    "**/test-results.xml": "",
+    "**/test-results.browser.xml": ""
+  },
+  "matrix": {
+    "Agent": {
+      "windows-2019": { "OSVmImage": "MMS2019", "Pool": "azsdk-pool-mms-win-2019-general" },
+      "ubuntu-18.04": { "OSVmImage": "MMSUbuntu18.04", "Pool": "azsdk-pool-mms-ubuntu-1804-general" }
+    },
+    "Scenario": {
+      "coverage": { "PublishCodeCoverage": "true", "TestType": "node", "TestResultsFiles": "**/test-results.xml" },
+      "sample": { "TestType": "sample" },
+      "browser": { "TestType": "browser", "TestResultsFiles": "**/test-results.browser.xml" }
+    },
+    "NodeTestVersion": [ "8.x", "10.x", "12.x" ]
+  },
+  "include": [
+    {
```

Comment:
It feels like TestType=node should just be another Scenario in the primary matrix. I realize there are some slight differences (i.e. node 8/14) but I have to wonder if those are on purpose or someone just not wanting to add the full set of entries.

Reply:
It should, but I couldn't add it in without a significant delta from the current test matrix, since it's configured to test all node versions against ubuntu. The sparse matrix wrapping gets messed up if I combine it. I'd like to iterate to simplify the matrix (as there are some requirements which may or may not be important, like windows for browser tests), but wanted to keep the test config as close to what's in master right now for the first big change.

Reply:
I like the idea of having a smaller delta but we should also way that against simplifying and making these more consistent across the repo. I'm good with either option I mostly wanted to call out that it feels like this entry looks off.

Reply:
We found a simplified version of the matrix.